### DISCUSSION TITLE
Open in safari in current tab group

### DISCUSSION THREAD

**Discussion by ajmarroquin** [will-stone/browserosaurus#568] [Q&A] [answered]
First off, I love browserosaurus, i've tried a few other link routing apps and haven't fallen in love with any others.

Second, it seems to me that when i select Safari as the browser to open, it always opens a new window, never the window that i currently have open. This seems like its due to the fact that i utilize safari's "tab group" option. Even without the ability to switch WHICH tab group it opens in, i'd love to have browserosaurus open the link in the currently active tab group. 

Third, which falls into feature request, it would be great if when selecting safari it gave you the option to select which tab group, although most often it would be the active group so i dont put that in a high priority. 

B version: V 19.1.0

https://user-images.githubusercontent.com/43194782/189177361-45f0905e-bb2e-405e-aab8-2455648c9477.mov

**Comment by will-stone** [comment, marked as answer]
Hi @ajmarroquin Okay, so the new window is only opened if you are currently inside a group; if you are currently looking at non-grouped tabs then it'll open in the same window. It looks like this is default Safari behaviour, by that I mean, if you set Safari as your default browser (instead of Browserosaurus) then it'll do this too. I'd rather not spend any time looking to tweak this behaviour as people may actually prefer it, it being an Apple default. Maybe it's best to log this with Apple as they may provide configuration that can then be inherited by Browserosaurus.

As for the feature request: I don't use Safari and therefore its tab grouping feature, but I'd be happy to review a pull request if you'd like to submit one.

**Reply by ajmarroquin** [reply]
@will-stone I agree with your assessment here. Not worth trying to fix something that is a default behavior of safari. I also gave up on tab groups after having tons of other issues with iCloud sync and tab groups. It's just not developed well yet on the Apple side. Thanks again for the great product and reply!
